e2e/features/steps/login/common.steps.ts

```diff
@@ -0,0 +1,20 @@
+import { Given, Then } from "@cucumber/cucumber";
+import { expect, page } from '../../../utils/index'
+import { baseUrl } from "../../../utils/consts";
+
+
+Given('the user is on the login page', async function () {
+    await page.goto(`${baseUrl}`)
+});
+
+Given('the user clicks on the {string} button', async function (text: string) {
+    //There should be a better way to do this, but there might be an issue with the mat-button selector
+    //Should be the same as in the ../register/common.steps.ts
+    const buttonSelector = `#main-content > ion-router-outlet > app-login > ion-content > ion-grid > ion-row > ion-col > ion-card > ion-card-content > ion-button`
+    const selector = await page.waitForSelector(buttonSelector);
+    await selector.click();
+});
+
+Then('the page should navigate to the register page', async function () {
+    expect(page.url()).toBe(`${baseUrl}/register`)
```

Comment:
For me the e2e test throws the following error:
```
1) Scenario: User navigates to register # features/login.feature:3
   ✔ Given the user is on the login page # features/steps/login/common.steps.ts:6
   ✔ And the user clicks on the "Még nincs fiókja?" button # features/steps/login/common.steps.ts:10
   ✖ Then the page should navigate to the register page # features/steps/login/common.steps.ts:18
       Error: expect(received).toBe(expected) // Object.is equality
       
       Expected: "http://localhost:8080/register"
       Received: "http://localhost:8080/login"
           at Proxy.<anonymous> (/workspaces/sk0/e2e/node_modules/playwright/lib/matchers/expect.js:174:37)
           at World.<anonymous> (/workspaces/sk0/e2e/features/steps/login/common.steps.ts:19:24)
           at /workspaces/sk0/e2e/features/steps/login/common.steps.ts:8:71
           at __awaiter (/workspaces/sk0/e2e/features/steps/login/common.steps.ts:4:12)
           at World.<anonymous> (/workspaces/sk0/e2e/features/steps/login/common.steps.ts:30:12)
```

If I use ```await page.waitForUrl(`${baseUrl}/register`)``` then it works.

Before running the test I follow the docker compose up + npm i (in e2e) method.

Reply:
Worked for me without it as well, but added to both place where we are checking the stat of the url.

Reply:
Thats the other solution
``` await page.waitForLoadState("networkidle")```